pkg/daemon/ceph/client/pool.go

```diff
@@ -215,6 +216,28 @@ func CreatePoolForApp(context *clusterd.Context, clusterName string, newPool Cep
 	return nil
 }
 
+func createReplicationCrushRule(context *clusterd.Context, clusterName string, newPool CephStoragePoolDetails, ruleName string) error {
+	if newPool.FailureDomain == "" {
```

Comment:
does this mean that in order to specify an alternate crush root, the user must also provide a failure domain?  if so, is that documented explicitly anywhere?

Reply:
i'll fix that, it was not intended